lib/TPP/DefaultTppPasses.cpp

```diff
@@ -91,18 +91,7 @@ struct DefaultTppPasses : public DefaultTppPassesBase<DefaultTppPasses> {
     pm.addPass(bufferization::createEmptyTensorToAllocTensorPass());
 
     // Run bufferization as the rest of the passes prefer working on memref.
-    bufferization::OneShotBufferizationOptions buffOpts;
-    buffOpts.allowReturnAllocs = true;
-    buffOpts.bufferizeFunctionBoundaries = true;
-    buffOpts.functionBoundaryTypeConversion =
-        bufferization::LayoutMapOption::IdentityLayoutMap;
-    pm.addPass(bufferization::createOneShotBufferizePass(buffOpts));
-    pm.addPass(bufferization::createDropEquivalentBufferResultsPass());
-    pm.addNestedPass<func::FuncOp>(
-        bufferization::createFinalizingBufferizePass());
-    // Clean up after bufferization.
-    pm.addPass(bufferization::createBufferDeallocationPass());
```

Comment:
This should be added either to the bufferization pass or to the default pipeline after bufferization.

Reply:
is the post-processing in Bufferize.cpp

Reply:
Probably the default pipeline. But tests seems happy so I can add it separately with some test case.

Reply:
Nevermind, I'm just blind. Another `createBufferDeallocationPass` is already present at the end of the default pipeline and there is a test case in `DefaultPipeline/mixed.mlir` that verifies buffer deallocation.
No need for any changes.